lib/CleantalkSP/SpbctWP/Settings/DomainExclusion.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace CleantalkSP\SpbctWP\Settings;
+
+use CleantalkSP\SpbctWP\Scanner;
+use CleantalkSP\SpbctWP\Helpers\CSV;
+
+class DomainExclusion
+{
+    private $DOMAIN_EXCLUSION_FILE_NAME = 'spbct_allowed_domains.txt';
+
+    public function domainExclusionsView($exclusions)
```

Comment:
Rename method and functions from domainExclusions to frontendScanDomainExclusions. This is not obvious, what the class used for without docs.